dotCMS/src/integration-test/java/com/dotcms/enterprise/publishing/remote/bundler/ContainerBundlerTest.java

```diff
@@ -103,31 +104,38 @@ public void addContainerInBundle(final TestCase testCase)
         final FilterDescriptor filterDescriptor = new FilterDescriptorDataGen().nextPersisted();
 
         final PushPublisherConfig config = new PushPublisherConfig();
-        config.add(container, PusheableAsset.CONTAINER);
-        config.setOperation(PublisherConfig.Operation.PUBLISH);
 
-        final DirectoryBundleOutput directoryBundleOutput = new DirectoryBundleOutput(config);
+        try (ManifestBuilder manifestBuilder = new TestManifestBuilder()) {
```

Comment:
Add to the expected result that's also checking manifest logic

Reply:
done https://github.com/dotCMS/core/pull/20632/commits/562949d0d780b11ebcf577a03de992b3d2d150dc